src/sentry/models/integrations/sentry_app_installation.py

```diff
@@ -33,6 +36,22 @@ def get_installed_for_organization(self, organization_id: int) -> QuerySet:
     def get_by_api_token(self, token_id: str) -> QuerySet:
         return self.filter(status=SentryAppInstallationStatus.INSTALLED, api_token_id=token_id)
 
+    def get_projects(self, token: ApiToken) -> QuerySet[Project]:
+        from sentry.models import Project, SentryAppInstallationToken
+
+        try:
+            installation = self.get_by_api_token(token.id).first()
```

Comment:
`.first()` will apply an order clause to the query. Do you want that or would `.get()` work? The benefits of `.get()` are that it will raise an error if there are duplicate tokens (which I hope there are not).